golang/lib/services/shared_path.go

```diff
@@ -0,0 +1,39 @@
+package services
+
+import (
+	"path/filepath"
+)
+
+/*
+Represents and absolute path shared between two containers, it could be a file, a directory or a symlink
```

Comment:
Recommended rephrasing - "Holds information about a filepath shared between two containers: this container, and a container running a service in a testnet. The actual object referenced by this path could be anything - a fire, a directory, a symlink, nonexistent, etc."

Second: this documentation should definitely be in the `lib-documentation` file, and replaced with a `Docs available at.....` link